static/app/views/dashboards/datasetConfig/spans.tsx

```diff
@@ -120,7 +121,10 @@ export const SpansConfig: DatasetConfig<
       pageFilters,
       limit,
       cursor,
-      referrer
+      referrer,
+      undefined,
+      undefined,
+      samplingMode
```

Comment:
Unrelated to this PR, but we should probably consider making this a object instead of passing in `undefined` multiple times

Reply:
Totally, it's on my TODO list to change these to objects because ugh lol every time someone has to add one, you need to do this, but I'll leave it off for another PR

Reply:
100% it should be handled separately imo